cli-plugins/socket/socket.go

```diff
@@ -0,0 +1,72 @@
+package socket
+
+import (
+	"errors"
+	"io"
+	"net"
+	"os"
+
+	"github.com/docker/distribution/uuid"
+)
+
+// EnvKey represents the well-known environment variable used to pass the plugin being
+// executed the socket name it should listen on to coordinate with the host CLI.
+const EnvKey = "DOCKER_CLI_PLUGIN_SOCKET"
```

Comment:
cc @milas @glours FYI this const was moved; in case code changes are needed when updating the module in compose